Tests/TestRunners/MultiFileTestFixture.cs

```diff
@@ -37,7 +37,7 @@ public sealed class MultiFileTestFixture : ICollectionFixture<MultiFileTestFixtu
         /// <summary>
         /// Turn it and run the test, then you can manually check the output loads/builds in VS.
         /// </summary>
-        private readonly bool _writeAllFilesForManualTesting = false;
+        private readonly bool _writeAllFilesForManualTesting = true;
```

Comment:
Ah yes. I turned it on to add the new Prefix.Vblibrary project to the expected solution files in the characterization tests, but it added all the other project files from the sourcefiles as well. I'll manually delete the unnecessary ones again, since there is some perfomance penalty indeed and for the tests to be more concise/clear.